Update "Enabling the Ruby profiler" to use the new `ddprofrb` command (#22037)
DataDog/documentation: content/en/profiler/enabling/ruby.md

2. Add the `ddtrace` gem to your `Gemfile` or `gems.rb` file:

    ```ruby

>     gem 'ddtrace', '~> 1.20'

Reviewer: Should this not reflect `1.21`? The version in which this change is introduced?

Author: Fixed in https://github.com/DataDog/documentation/pull/22037/commits/5cb29c05195ef3813bba130e385fe19c69cb17cf

>     gem 'ddtrace', '~> 1.20'

Author: ```suggestion
    gem 'ddtrace', '~> 1.21'
```

{{% /tab %}}
{{< /tabs >}}

4. Add the `ddprofrb exec` command to your Ruby application start command:

    ```shell
    bundle exec ddprofrb exec ruby myapp.rb
    ```

    Rails example:

    ```shell
    bundle exec ddprofrb exec bin/rails s
    ```

    If you're running a version of `ddtrace` older than 1.21.0, replace `ddprofrb exec` with `ddtracerb exec`.

    **Note**

>     If starting the application via `ddprofrb exec` is not an option (eg. when using the Phusion Passenger web server), you can alternatively start the profiler by adding the following to your application entry point such as `config.ru` for a web application:

Reviewer 2: ```suggestion
    If starting the application with `ddprofrb exec` is not an option (for example, when using the Phusion Passenger web server), you can alternatively start the profiler by adding the following to your application entry point (such as `config.ru`, for a web application):
```
